import_export_extensions/admin/model_admins/export_job_admin.py

```diff
@@ -1,4 +1,6 @@
 
+from http import HTTPStatus
```

Comment:
```suggestion
import http

http.HTTPStatus.NOT_FOUND
```
Just personal preference